Cargo.toml

```diff
@@ -27,6 +27,9 @@ cc = "1.0"
 
 [dependencies]
 raw-window-handle = "0.3.3"
+cast = "0.2"
+time = "0.1.34"
+instant = { version = "0.1.9", features = [ "web-sys", "now" ] }
 
```

Comment:
Are these required as general dependencies?

Reply:
no, I have to clean up this. I don't know why time and cast are here at all. I know I need instant for the web part, but definitely not a general dependency. I'll move it to the web dependencies later.

Reply:
A little late, but I removed the dependency from the global section and it's now only required when building the web part